### ISSUE TITLE
Review scope

### ISSUE THREAD

**Issue by raq929** [ga-wdi-boston/js-context-this#38]
Some people have forgotten the scope lesson from the first unit at this point. Starting the talk with a short review of scope might be helpful.

**Comment by jrhorn424** [comment]
> Starting the talk with a short review of scope might be helpful.

What's the only way to create a new scope in JS? A function!

Review done. 😛

**Comment by raq929** [comment]
I'm pretty sure they needed 'What is scope again?' at least briefly.

**Comment by MicFin** [comment]
For cohort 016 we introduced block scope and closures in the `functions-ins-and-outs` (after presenting, we discussed breaking that lesson up into 2 because it was a lot for 1 lesson).  We did not mention function scope other than how using `var` destroys our block scope expectations so don't use `var`. 

Is there good reason to introduce function scope here?  
Should we do a similar review on scope as we did in `js-functions-in-and-outs`?  
Do we no longer need to review scope since we covered it already?
